analysis.R

```diff
@@ -0,0 +1,222 @@
+      Xmarker10_count<-Xmarker10_count + 1 
+    }}
+  
+  for (row in 1:nrow(YallData)){
+    if (YallData$marker01[row] == 1){
+      Ymarker01_count<-Ymarker01_count + 1 
+    }
+    if (YallData$marker02[row] == 1){
+      Ymarker02_count<-Ymarker02_count + 1 
+    }
+    if (YallData$marker03[row] == 1){
+      Ymarker03_count<-Ymarker03_count + 1 
+    }
+    if (YallData$marker04[row] == 1){
+      Ymarker04_count<-Ymarker04_count + 1 
+    }
+    if (YallData$marker05[row] == 1){
+      Ymarker05_count<-Ymarker05_count + 1 
+    }
+    if (YallData$marker06[row] == 1){
+      Ymarker06_count<-Ymarker06_count + 1 
+    }
+    if (YallData$marker07[row] == 1){
+      Ymarker07_count<-Ymarker07_count + 1 
+    }
+    if (YallData$marker08[row] == 1){
+      Ymarker08_count<-Ymarker08_count + 1 
+    }
+    if (YallData$marker09[row] == 1){
+      Ymarker09_count<-Ymarker09_count + 1
+    }
+    if (YallData$marker10[row] == 1){
+      Ymarker10_count<-Ymarker10_count + 1 
+    }}
+  
+  #create dataframes for marker information
+  Xmarker.df <- data.frame(matrix(NA,10,3))
+  colnames(Xmarker.df) <- c("country", "marker","occurrence")
+  Xmarker.df[,1] <- "X"
+  Xmarker.df[,2] <- c("marker01","marker02","marker03","marker04","marker05","marker06","marker07","marker08","marker09","marker10")
+  Xmarker.df[,3] <- c(Xmarker01_count,Xmarker02_count,Xmarker03_count,Xmarker04_count,Xmarker05_count,Xmarker06_count,Xmarker07_count,Xmarker08_count,Xmarker09_count,Xmarker10_count)
+  
+  Ymarker.df <- data.frame(matrix(NA,10,3))
+  colnames(Ymarker.df) <- c("country", "marker","occurrence")
+  Ymarker.df[,1] <- "Y"
+  Ymarker.df[,2] <- c("marker01","marker02","marker03","marker04","marker05","marker06","marker07","marker08","marker09","marker10")
+  Ymarker.df[,3] <- c(Ymarker01_count,Ymarker02_count,Ymarker03_count,Ymarker04_count,Ymarker05_count,Ymarker06_count,Ymarker07_count,Ymarker08_count,Ymarker09_count,Ymarker10_count)
+  
+  #combine X Y marker dataframes
+  allMarkers <- rbind(Xmarker.df, Ymarker.df)
+  
+  #create the bargraph to compare marker occurrence for each country
+  library(ggplot2)
+  
+  ggplot(allMarkers, aes(x = marker, y = occurrence, fill = country)) +
+    geom_bar(stat="identity", position = position_dodge(), alpha = 0.75) +
+    theme_classic() +
+    xlab("Marker Number") +
+    ylab("Occurrence") +
+    theme(axis.text.x = element_text(angle=65, vjust=0.6))
+  #CountryX markers are labeled in red
+  #CountryY markers are labeled in blue
+}
+
+Marker.Bar("allData.csv")
+#Since each country has different representative markers 
+#(CountryX is predominantly markers 01-05, and 
+#CountryY is predominantly markers 06-10), a vaccine developed
+#in CountryY would not likely work for CountryX because each country 
+#will have different immunological responses.
+#If a vaccine is developed in CountryY, it will most likely target markers 06-10,
+#and very little people in CountryX have markers 06-10.
```

Comment:
Good commenting and efficient code that uses coding concepts that we covered in class – 3.75 points (4 is max)